integration_tests/pages/bookedtoday/arrivals/matchingRecords/possibleRecordsFound.ts

```diff
@@ -0,0 +1,28 @@
+import Page, { PageElement } from '../../../page'
+
+export default class PossibleRecordsFoundPage extends Page {
+  constructor() {
+    super('Possible existing records have been found')
+  }
+
+  static goTo(id: string): PossibleRecordsFoundPage {
+    cy.visit(`/prisoners/${id}/possible-records-found`)
+    return Page.verifyOnPage(PossibleRecordsFoundPage)
+  }
+
+  perElement = (element): PageElement => cy.get(`.data-qa-per-record-${element}`)
+
+  matchingElement = (element): PageElement => cy.get(`.data-qa-matching-record-${element}`)
```

Comment:
If you have multiple matches would you end up with each match having the same class?

Reply:
There is only one instance of these classes in the page because the 'element' will be passed in the loop index which would be unique